src/app/js/admin/ImportModelButton.js

```diff
@@ -22,31 +27,59 @@ const useStyles = makeStyles({
     buttonText: {
         textTransform: 'none',
     },
+    input: {
+        position: 'absolute',
+        top: 0,
+        left: 0,
+        right: 0,
+        bottom: 0,
+        opacity: 0,
+        width: '100%',
+        cursor: 'pointer',
+    },
 });
 
 export const ImportModelButtonComponent = ({
     className,
     importFieldsClosed,
     hasPublishedDataset,
     p: polyglot,
+    nbFields,
+    importFields,
+    succeeded,
+    failed,
 }) => {
     const classes = useStyles();
-    const [open, setOpen] = useState(false);
     const [
         showImportFieldsConfirmation,
         setShowImportFieldsConfirmation,
     ] = useState(false);
-    const [showSuccessAlert, setShowSuccessAlert] = useState(false);
 
-    const handleImportFieldsClose = sucess => {
+    const [showAlert, setShowAlert] = useState(false);
+    const [useInput, setUseInput] = useState(false);
```

Comment:
Careful with this kind of naming in react, we could think it's a hook (and not the result of a hook)